src/org/usfirst/frc/team467/robot/UltrasonicSensor.java

```diff
@@ -29,8 +27,7 @@ public static UltrasonicSensor getInstance() {
 	}
 	
 	public double getDistance() {
-		distance = us.getRangeInches();
-		return distance;
+		return us.getRangeInches();
```

Comment:
This has nothing to do with the Grabber, but these are benign and simple so you can keep them.